README.md

```diff
@@ -15,6 +15,41 @@ This project is under active development. The workflow code is being piloted in
 
 More documentation and samples will be coming soon!
 
+## Getting the library
+
+### Swift
+
+[![CocoaPods compatible](https://img.shields.io/cocoapods/v/Workflow.svg)](https://cocoapods.org/pods/Workflow)
+
+If you use CocoaPods to manage your dependencies, simply add Workflow and WorkflowUI to your Podfile:
+
+```ruby
+pod 'Workflow'
+pod 'WorkflowUI'
+```
+
+### Kotlin
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.squareup.workflow/workflow-core.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.squareup.workflow%22)
+
+Artifacts are hosted on Maven Central. If you're using Gradle, ensure `mavenCentral()` appears in your `repositories` block, and then add dependencies on the following artifacts:
+
+| Subproject | Depend on this if… | Maven Coordinates |
```

Comment:
This table is a bit wide, and so hard to read. WDYT of dropping the headers and making it just two cells, moving the coordinates to the left, under the names:

| Subproject        | Depend on this if… | 
|-------------------|--------------------|
| workflow-core<br>`com.squareup.workflow:workflow-core:${workflowVersion}`| You are writing a library module/project that uses Workflows, but you don't need to interact with the runtime from the outside. | 
| workflow-rx2<br>`com.squareup.workflow:workflow-rx2:${workflowVersion}` | You need to interact with RxJava2 from your Workflows. | 
| workflow-testing<br>`com.squareup.workflow:workflow-testing:${workflowVersion}` | You are writing tests. This should only be included as a test dependency. | 
| workflow-ui-android<br>`com.squareup.workflow:workflow-ui-android:${workflowVersion}` | You're writing an Android app that uses Workflows. |

Reply:
Shoot, the coordinates are still wrapping, which is what bothered me most.

Reply:
Also might be nicer as a simple dependencies block, with inline comments for the "depend on this" bit.

```
dependencies {
  // When writing a library module/project that uses Workflows, with no need to interact with the runtime from the outside.
  implementation com.squareup.workflow:workflow-core:${workflowVersion}

 // etc...
}
```

Reply:
Fixed the tables. Github supports basic HTML table syntax, including the `nowrap` attribute, so they wrap correctly now.

Reply:
I would like to avoid the giant code block, it's harder to scan.